addon/components/au-data-table/text-search.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="au-c-data-table__search">
-  {{input value=value placeholder=placeholder class="au-c-input au-c-input--block"}}
+  <Input value={{this.value}} placeholder={{this.placeholder}} class="au-c-input au-c-input--block" />
```

Comment:
> Is using @ because of 2-way binding only? The placeholder is using this. and it's getting the correct values.

Yes, at the moment typing in the input field will not update `this.value`. It would be fine if there was an action attached to the input event that does the updating but that isn't the case here. So I assume this depended on the 2-way binding to work.

> I don't see this component constructed with a `@value` argument, so where does this value come from anyway?

It extends the component from the ember-data-table addon which [sets the value on init (and in some other places)](https://github.com/mu-semtech/ember-data-table/blob/eb1ec672c34b5131ba4048f60670d743606cb61b/addon/components/text-search.js#L17). I haven't used the component myself so I'm not sure what issue you mean.

Reply:
Okay, nevermind. Small misunderstanding on my part. Using `@value` (in the correct place) indeed solves all of my problems.